### PR TITLE
Rc 0.5.4

### PR DESCRIPTION
**Pull request by AndyGura**

### CODE REVIEW

File: .github/workflows/tag_action.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up environment variables
+        run: |
+          echo "NOTION_DATABASE_ID=e3aca840563b4cc8b273e65da748e1dd" >> $GITHUB_ENV
+          echo "WEBHOOK_URL=https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e" >> $GITHUB_ENV
```
Comment (timohanke) [inline .github/workflows/tag_action.yml:22]: We have these definitions in the `env:` section at the top already. Can be removed?

**Review by timohanke — COMMENTED** [review]
1 comment about tag_action

**Comment by timohanke** [comment]
In the README please change under copyright: MR Research 2023-24
(the year -24 is added)

**Comment by timohanke** [comment]
In CI please change these lines:
```
        uses: actions/checkout@v4.1.7
      - uses: ZenVoich/setup-mops@v1.2
```
(latest versions)

File: .github/workflows/tag_action.yml (hunk 1 above)
Comment (AndyGura) [reply .github/workflows/tag_action.yml]: removed

**Review by dpanchenko — APPROVED** [review]

**Pull request merged**